### PR TITLE
Handle invalid vehicle IDs

### PR DESCRIPTION
**Pull request by DylanVanAssche**
Handle invalid vehicle IDs like S trains, ICL trains and ICT trains.
Mapping:

- S trains are actually L trains
- ICT and ICL trains are actually IC trains

### CODE REVIEW

File: connections.py [hunk 1]
```diff
@@ -63,14 +64,33 @@ def get_buckets(self, vehicle_id, data_type, station):
             for b in bucket_list:
                 bucket_list[b] = round(100 * (bucket_list[b] / number_of_entries), PRECISION)
 
+            print(bucket_list)
```
Comment (juliavdkris) [inline connections.py]: Only for debugging, we should remove this

File: connections.py [hunk 2]
```diff
@@ -63,14 +64,33 @@ def get_buckets(self, vehicle_id, data_type, station):
...
             return bucket_list
         else:
             return None
 
+    def handle_invalid_vehicle_id(self, vehicle_id):
+        # S trains are mapped on L trains in the data
+        # Train number are always the last 4 digits, but we have S51 and S1
+        # trains for example
+        if 'S' in vehicle_id:
+            vehicle_id = 'L' + vehicle_id[-4:]
```
Comment (juliavdkris) [inline connections.py:76]: Would be cleaner to do `.replace('S', 'L')` in case there are IDs with different lengths (and for future proofing)

**Review by juliavdkris — CHANGES_REQUESTED** [review]
Some small changes but other than that, LGTM

File: connections.py (hunk 2 above)
Comment (DylanVanAssche) [reply connections.py:76]: S trains have a number, this 'feature' doesn't allow us to do it with `replace`

**Review by juliavdkris — APPROVED** [review]
LGTM

**Pull request merged**